int16 color  #敌方颜色 0：红色 1：蓝色
int16 remaining_time     #剩余比赛时间
int16 blue_base_protected #蓝方基地无敌
int16 blue_base_shield   #蓝方基地护盾
int16 blue_base_blood    #蓝方基地血量
int16 red_base_proteted  #红方基地无敌
int16 red_base_shield    #红方基地护盾
int16 red_base_blood     #红方基地血量
int16 r1_blood  #红英雄血量
int16 r1_bullet #红英雄弹量
int16 b1_blood  #蓝英雄血量
int16 b1_bullet #蓝英雄弹量
int16 r3_blood  #红3号步兵血量
int16 r3_bullet #红3号步兵弹量
int16 b3_blood  #蓝3号步兵血量
int16 b3_bullet #蓝3号步兵弹量
int16 r4_blood  #红4号步兵血量
int16 r4_bullet #红4号步兵弹量
int16 b4_blood  #蓝4号步兵血量
int16 b4_bullet #蓝4号步兵弹量
int16 r7_blood  #红哨兵血量
int16 r7_bullet #红哨兵弹量
int16 b7_blood  #蓝哨兵血量
int16 b7_bullet #蓝哨兵弹量
int16 benefit_area_open #中心增益点是否打开 0:关闭 1:打开
int16 blue_energy  #蓝方能量
int16 red_energy   #红方能量
#枪口热量
#弹速
#机器人等级
#